react-components/src/components/RevealContainer/RevealContainer.tsx

```diff
@@ -7,10 +7,13 @@ import { Cognite3DViewer, type Cognite3DViewerOptions } from '@cognite/reveal';
 import { RevealContext } from './RevealContext';
 import { type Color } from 'three';
 import { ModelsLoadingStateContext } from '../Reveal3DResources/ModelsLoadingContext';
+import { SDKProvider } from './SDKProvider';
+import { QueryClientProvider, QueryClient } from '@tanstack/react-query';
 
 type RevealContainerProps = {
   color?: Color;
   sdk: CogniteClient;
+  uiElements?: ReactNode;
```

Comment:
This was added because before if you want to access contexts provided by `RevealContainer`, any element should be a child of Reveal main `domElement` which results in these elements being reactive to reveal clicks/wheel events messing up camera controls. The same applies for the toolbar but it should be addressed in a separate PR. Current version of toolbar component added in the same way as in the story will break camera controls if you enable `cameraTargetOnClick` or any other interaction with reveal based on click events from the viewer.